Aptoide/appcoins-wallet-android
# [MOB-168] Subs MTP integration

**What does this PR do?**

Integrates the MTP for subscriptions

**Database changed?**

   No

**Where should the reviewer start?**

AppcoinsBillingBinder
SubscriptionsBillingApi
RemoteRepository

**How should this be manually tested?**

Use the Trivial Drive from branch 168 - [Branch](https://github.com/Aptoide/appcoins-iab-sample/tree/feature/MOB-168-Subs-Trivial-drive)
Try to make an inapp purchase with every method in both managed and unmanaged payments to check if nothing broke
Try to make a subs purchase with CC and Paypal (Use the trivial drive version that I'll put in the ticket when it's ready)

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/MOB-168

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new libs, new keys, etc.)




**Code Review Checklist**
